localized_data/text_data.json

```
@@ -12267,22 +12267,22 @@
 		"1000138": "【橡树大赛】育成时在樱花赏进入前5名",
 		"1000139": "【橡树大赛】育成时在橡树大赛（オークス）\n取胜",
 		"1000140": "【橡树大赛】完成所有限定任务",
-		"1000141": "【日本ダービー】育成を1回完了しよう",
-		"1000142": "【日本ダービー】育成を3回完了しよう",
-		"1000143": "【日本ダービー】育成を5回完了しよう",
-		"1000144": "【日本ダービー】育成を10回完了しよう",
-		"1000145": "【日本ダービー】育成で青葉賞の2着以内に入ろう",
-		"1000146": "【日本ダービー】育成で皐月賞の5着以内に入ろう",
-		"1000147": "【日本ダービー】育成で日本ダービーに勝利しよう",
-		"1000148": "【日本ダービー】限定ミッションをすべてクリアしよう",
-		"1000149": "【安田記念】育成を1回完了しよう",
-		"1000150": "【安田記念】育成を3回完了しよう",
-		"1000151": "【安田記念】育成を5回完了しよう",
-		"1000152": "【安田記念】育成を10回完了しよう",
-		"1000153": "【安田記念】育成でマイラーズカップを1着で勝利しよう",
-		"1000154": "【安田記念】育成で京王杯スプリングカップを1着で勝利しよう",
-		"1000155": "【安田記念】育成で安田記念に勝利しよう",
-		"1000156": "【安田記念】限定ミッションをすべてクリアしよう",
+		"1000141": "【日本德比】完成1次育成",
+		"1000142": "【日本德比】完成3次育成",
+		"1000143": "【日本德比】完成5次育成",
+		"1000144": "【日本德比】完成10次育成",
+		"1000145": "【日本德比】育成で青葉賞の2着以内に入ろう",
+		"1000146": "【日本德比】育成で皐月賞の5着以内に入ろう",
+		"1000147": "【日本德比】育成で日本德比に勝利しよう",
```

比赛名和技能名建议不要批量替换，在任务要求中保留中日双语。
```suggestion
		"1000147": "【日本德比】育成时在日本德比（日本ダービー）\n取胜",
```

---

已按照建议更改进行逐条手动补足。但审查其余字段时发现，既有字段对此种内容的表述存在句式、用词不一的情况。待后续进行更改。